apps/src/aiComponentLibrary/suggestedPrompt/SuggestedPrompt.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import moduleStyles from './suggested-prompt.module.scss';
+
+/**
+ * Renders a clickable tag that can be customized with a suggested prompt.
+ */
+
+export interface SuggestedPromptProps {
+  onClick: () => void;
+  label: string;
+  show: boolean;
+}
+
+const SuggestedPrompt: React.FunctionComponent<SuggestedPromptProps> = ({
+  onClick,
+  label,
+  show,
+}) => (
+  <>
+    {show ? (
+      <div onClick={onClick} className={moduleStyles.prompt}>
+        <span>{label}</span>
+      </div>
+    ) : null}
+  </>
+);
```

Comment:
Not blocking, but I'd suggest moving the conditional outside of the empty block so we can get rid of it entirely.

Further, I'd also recommend inverting the logic so we can cuddle the `null` case more closely to the top and minimize it getting lost with distance from the check.

```
=> (
  !show
    ? null
    : <div onClick={onClick} className={moduleStyles.prompt}>
        <span>{label}</span>
      </div>
```

(or something like that)